MovieApp/Assets.xcassets/ironmanposter.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "ironmanposter.jpeg",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
```

Comment:
When exporting a bitmap image from Figma, you have an option to export different scales. Always add @1x, @2x and @3x resolutions so that the image can be rendered properly across all iOS devices.